source/includes/_disputes.md

```diff
@@ -1160,6 +1160,69 @@ You will also need to attach the Braintree transaction id using the `charge` par
 
 You can always reconnect your Braintree account from the settings page [here](/dashboard/settings/processors) to grant Chargehound access to the disputes API, this will make your integration easier.
 
+## Stripe listing needs response
```

Comment:
I think this should be in the integration section for responding to backlog. It should explain that to make sure you get disputes and retrievals to use them both